tmt/steps/provision/__init__.py

```diff
@@ -1103,6 +1109,59 @@ def requires(cls) -> list['tmt.base.Dependency']:
         """ All requirements of the guest implementation """
         return []
 
+    def acquire_log(self, log_name: str) -> Optional[str]:
+        """
+        Fetch and return content of a log.
+        :param log_name: name of the log.
+        :returns: content of the log.
```

Comment:
The `Optional` deserves to be explained as well: when would `None` be returned? E.g. "content of the log, or `None` if the log cannot be retrieved"?